packages/wdio-mocha-framework/src/constants.ts

```diff
@@ -1,9 +1,15 @@
 export const INTERFACES = {
-    bdd: ['it', 'before', 'beforeEach', 'after', 'afterEach'],
+    bdd: ['it', 'specify', 'before', 'beforeEach', 'after', 'afterEach'],
```

Comment:
I see that I forgot to mention that it's also missing `context` in this list, mind adding this as well so the list is complete?

Reply:
context is not a test interface. It is treated similar to describe. As describe isn't in this list, context shouldn't be a part of this list as well in my opinion..correct me if I am wrong though.

Reply:
You're absolutely right. Gotta love switching context from work to an open source project hehe, sorry for overlooking that.